### ISSUE TITLE
TypeError: Can't instantiate abstract class PlaybookGrapherCLI with abstract methods init_parser, post_process_args

### ISSUE THREAD

**Issue by JakobP** [haidaraM/ansible-playbook-grapher#18]
When following the setup guide on MacOs 10.14 and Ubuntu 18.04 I get an abstract methods error. I installed on a new Ubuntu 18.04 machine when testing.

Tried with both Python 2.7 and Python 3.6

```
vagrant@ubuntu-bionic:/vagrant/provisioning$ ansible-playbook-grapher --help
Traceback (most recent call last):
  File "/home/vagrant/.local/bin/ansible-playbook-grapher", line 11, in <module>
    sys.exit(main())
  File "/home/vagrant/.local/lib/python2.7/site-packages/ansibleplaybookgrapher/cli.py", line 85, in main
    cli = PlaybookGrapherCLI(args)
TypeError: Can't instantiate abstract class PlaybookGrapherCLI with abstract methods init_parser, post_process_args

```

**Comment by haidaraM** [comment]
Which version of ansible are you using ?

**Comment by haidaraM** [comment]
I guess it's ansible 2.8. Was able to reproduce in Travis: https://travis-ci.org/haidaraM/ansible-playbook-grapher/builds/542466763

**Comment by ric79** [comment]
Same error

```
$ pip freeze | grep ansible-playbook-grapher
ansible-playbook-grapher==0.9.0

]$ ansible --version
ansible 2.8.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/home/xxxxx/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.5 (default, Jul 13 2018, 13:06:57) [GCC 4.8.5 20150623 (Red Hat 4.8.5-28)]

$ more prova.yml
- name: Create nodeX
  hosts: localhost

  tasks:

   - debug:
       msg="CIAO"

$ ansible-playbook-grapher prova.yml
Traceback (most recent call last):
  File "/usr/bin/ansible-playbook-grapher", line 11, in <module>
    sys.exit(main())
  File "/usr/lib/python2.7/site-packages/ansibleplaybookgrapher/cli.py", line 85, in main
    cli = PlaybookGrapherCLI(args)
TypeError: Can't instantiate abstract class PlaybookGrapherCLI with abstract methods init_parser, post_process_args

```

**Comment by haidaraM** [comment]
Fixed in `0.9.1`.
